src/lib/header-bidding/a9/a9.spec.ts

```diff
@@ -104,3 +110,110 @@ describe('initialise', () => {
 		expect(window.apstag?.init).toHaveBeenCalled();
 	});
```

Comment:
Since adding my separate tests for this file, it is now producing this check failure for the tests not part of my changes.
I have tried type asserting and creating a mock type but to no avail. I'm not entirely sure what the issue is here.